core/tests/puppeteer-acceptance-tests/user-utilities/exploration-editor-utils.ts

```diff
@@ -872,6 +876,92 @@ export class ExplorationEditor extends BaseUser {
     }
     await this.clickOn(previewRestartButton);
   }
+
+  /**
+   * Function for creating an exploration with only EndExploration interaction with given title.
+   */
+  async createExplorationWithOnlyEndInteraction(
+    title: string
+  ): Promise<string | null> {
+    await this.navigateToCreatorDashboardPage();
+    await this.navigateToExplorationEditorPage();
+    await this.dismissWelcomeModal();
+    await this.createExplorationWithMinimumContent(
+      'Exploration intro text',
+      'End Exploration'
+    );
+    await this.navigateToSettingsTab();
+    await this.updateTitleTo(title);
+    await this.updateGoalTo('OppiaAcceptanceTestsCheck');
+    await this.selectCategory('Algebra');
+    await this.selectLanguage('Arabic');
+    await this.addTags(['TagA', 'TagB', 'TagC']);
+    await this.saveExplorationDraft();
+    return await this.publishExploration();
+  }
+
+  /**
+   * This function checks the number of subscribers in the Subscribers tab of the creator dashboard.
+   */
+  async expectNumberOfSubscribersToBe(subscriberCount: number): Promise<void> {
+    await this.page.waitForSelector(subscriberCountLabel);
+    const currentSubscriberCount = await this.page.$eval(
+      subscriberCountLabel,
+      element => element.textContent
+    );
+
+    if (
+      currentSubscriberCount &&
+      parseInt(currentSubscriberCount) === subscriberCount
+    ) {
+      showMessage(`Number of subscribers is equal to ${subscriberCount}`);
```

Comment:
We can have a full stop (.) at the end of sentence. 
Please review other strings/sentences also.

Reply:
Done.